### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by M00NJ**
I noticed some more things about the ui that could be improved. The number keypad in the timer tab doesn't use the material you color scheme. The timer and stopwatch icons in the navbar are swapped. The stopwatch should be the clock with the buttons on top. Also all the icons in the navbar are outlined except the stopwatch icon. The stopwatch circle could be a little larger since it's the only thing on screen in the stopwatch tab. Here are is a preview of the changes I made:

- added material you color scheme to number keypad
- changed color of backspace button to secondary container
- slightly increased button font size

numberpad

- fixed stopwatch and timer icons being swapped
- fixed stopwatch icon using .filled

navbar

- used bigger circle and font size in stopwatch tab
- change color of circle to fit better with material you color scheme

stopwatch-screen

- some other small things

When trying to fix the stopwatch icon using .filled, I noticed that all the navbar icons use .filled (.default), but only the stopwatch icon is actually drawn with fill and I couldn't figure out why. I fixed it by setting the stopwatch icon to .outlined, but there might be another way to do this that I don't know off.

Also when pushing the changes to my fork I must have made some newbie mistake, because there was another commit created with some translation updates. I don't know how to revert a this commit and cherry-picking the one I want to a new branch didn't work. But as far as I can tell, all the changes in that commit are already present in the main, so this shouldn't be a problem. If it is, please tell me how I can fix this. Hope you like the changes :)

### CODE REVIEW

File: app/src/main/java/com/bnyro/clock/ui/components/NumberKeypad.kt [hunk 1]
```diff
@@ -107,12 +111,13 @@ fun Button(
 
             onOperation(NumberKeypadOperation.AddNumber(number))
         },
-        modifier = Modifier.size(buttonSize)
+        modifier = Modifier.size(buttonSize),
+        color = buttonColor
```
Comment (Bnyro) [inline app/src/main/java/com/bnyro/clock/ui/components/NumberKeypad.kt:115]:
```suggestion
        color = MaterialTheme.colorScheme.surfaceColorAtElevation(1.dp)
```

File: app/src/main/java/com/bnyro/clock/ui/components/NumberKeypad.kt [hunk 2]
```diff
@@ -32,36 +34,37 @@ fun NumberKeypad(
...
     ) {
         Row(
             horizontalArrangement = Arrangement.spacedBy(buttonSpacing)
         ) {
-            Button(number = "1", buttonSize, onOperation)
-            Button(number = "2", buttonSize, onOperation)
-            Button(number = "3", buttonSize, onOperation)
+            Button(number = "1", buttonSize, buttonColor, onOperation)
+            Button(number = "2", buttonSize, buttonColor, onOperation)
+            Button(number = "3", buttonSize, buttonColor, onOperation)
```
Comment (Bnyro) [inline app/src/main/java/com/bnyro/clock/ui/components/NumberKeypad.kt:47]: Please remove the buttonColor arguments everywhere here.

**Comment by Bnyro** [comment]
And you can remove that merge commit by running `git rebase master` while being on your PR's branch. Afterwards, run `git push -f`.

**Comment by M00NJ** [comment]
I made the suggested changes. Makes a lot more sense to do it that way :)

**Comment by M00NJ** [comment]
> And you can remove that merge commit by running `git rebase master` while being on your PR's branch. Afterwards, run `git push -f`.

I was scared that this might override my local changes. I'm new to coding in general and thought that I could get away with focusing on learning Kotlin and how to use AndroidStudio and jetpackCompose, which is already quite a lot to take in. But I see now, that I will also have to learn how to use git properly, because if I don't know how this all works, I will probably make more mistakes.

**Comment by Bnyro** [comment]
You can test this without breaking anything by checking out to a new branch `git checkout -b my-test-branch-name` and then run `git branch` to confirm you're on a different branch. Anything you do in this branch won't affect the PR.
What you then want to do is
`git remote add upstream https://github.com/Bnyro/ClockYou
git fetch upstream
git rebase upstream/main`

**Comment by M00NJ** [comment]
> You can test this without breaking anything by checking out to a new branch `git checkout -b my-test-branch-name` and then run `git branch` to confirm you're on a different branch. Anything you do in this branch won't affect the PR. What you then want to do is `git remote add upstream https://github.com/Bnyro/ClockYou
> 
> git fetch upstream
> 
> git rebase upstream/main`

Thank you for taking the time to give me some useful tips. I highly appreciate it! :)

**Comment by Bnyro** [comment]
Please make sure to reset your branch/fork to be up to date with my repo before opening a new PR (can be done via the GitHub Ui)

**Comment by M00NJ** [comment]
Thank you. I have watched some videos about Git. I think the problem was, that my fork was setup as origin and upstream. Sorry for the messy PR. I will set this up correctly now so future PR won't look like this.

**Pull request merged**